QUEST_LV_0200.tsv

```diff
@@ -1529,37 +1529,37 @@ QUEST_LV_0200_20150317_001528	{memo X}Checking/LOOK_SIT/3/TRACK_BEFORE/None
 QUEST_LV_0200_20150317_001529	I like it sweet
 QUEST_LV_0200_20150317_001530	I'll get it
 QUEST_LV_0200_20150317_001531	Sowing is scary
-QUEST_LV_0200_20150317_001532	Secret of the Farmland (2)
+QUEST_LV_0200_20150317_001532	$Secret of the Farmland (2)
 QUEST_LV_0200_20150317_001533	I will take care of it
 QUEST_LV_0200_20150317_001534	Ignore it
 QUEST_LV_0200_20150317_001535	About the Seal
-QUEST_LV_0200_20150317_001536	Secret of the Farmland (1)
-QUEST_LV_0200_20150317_001537	I will activate the Guardian Stone
+QUEST_LV_0200_20150317_001536	$Secret of the Farmland (1)
+QUEST_LV_0200_20150317_001537	I will activate the Guardian Altar
 QUEST_LV_0200_20150317_001538	Let's just leave it
 QUEST_LV_0200_20150317_001539	{memo X}About the cure in Three Brooks Woods
 QUEST_LV_0200_20150317_001540	{memo X}Activating/SITABSORB/2/TRACK_BEFORE/None
-QUEST_LV_0200_20150317_001541	Power dewelling on the bee tree branch
+QUEST_LV_0200_20150317_001541	$Power Within the Bee Tree Branch
 QUEST_LV_0200_20150317_001542	I will collect the ashes
-QUEST_LV_0200_20150317_001543	Land with the Goddess' Powers(1)
+QUEST_LV_0200_20150317_001543	$Land Bestowed with the Goddess' Powers (1)
```

Comment:
I believe blessed would be a better fit here, as you're generally bestowed with blessings.